community/heirloom-doctools/APKBUILD

```diff
@@ -12,7 +12,8 @@ depends_dev=""
 makedepends="flex bison"
 install=""
```

Comment:
empty variables can be removed for brevity

Reply:
Thanks. PR has been updated to remove empty variables and builddir= decl